src/ui/public/stringify/types/String.js

```diff
@@ -55,10 +57,15 @@ export default function _StringProvider(Private) {
     }
   };
 
+  _String.prototype._toTitleCase = function (val) {
+    return val.replace(/\w\S*/g, txt => { return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase(); });
```

Comment:
The regex behaves more like what is expected from the (English) titlecase function:

    A Quick Brown Fox.	                    A Quick Brown Fox.
    STAY CALM!	                            Stay Calm!
    Mean, variance and standard_deviation.	Mean, Variance And Standard_deviation.

The lodash functions strip punctuation (including underscores) and do not process all uppercase words properly:

    A Quick Brown Fox.	                    A Quick Brown Fox
    STAY CALM!	                            STAY CALM
    Mean, variance and standard_deviation.	Mean Variance And Standard Deviation

I see that the lodash functions are used in `ui/public/filters/label.js`. Maybe the regex should be used there too?